auth/src/main/java/org/aerogear/auth/impl/UserPrincipalImpl.java

```diff
@@ -129,7 +119,9 @@ UserPrincipalImpl build() {
      */
     @Override
     public boolean hasRole(final IRole role) {
-        return roles.containsKey(role.getRoleName());
+//        TODO: to be implemented
+//        return roles.containsKey(role.getRoleName());
+        return false;
```

Comment:
If a method is not implemented, it is better to throw an exception than returning a wrong value.